hydromt/raster.py

```diff
@@ -1652,6 +1655,217 @@ def interpolate_na(self, method: str = "nearest", **kwargs):
         )
         return interp_array
 
+    def to_xyz(
+        self,
+        root: str,
+        px: int,
+        zoomlevels: list = [],
+        fmt: str = "tif",
+    ):
+        """Export rasterdataset to tiles in a xyz structure
+
+        Parameters
+        ----------
+        root : str
+            Path where the database will be saved
+            Database yml will be put one directory above
+        px : int
+            Amount of pixels per tile
+        zoomlevels : list, optional
+            Zoom levels to be put in the database
+
```

Comment:
There is not description of 'fmt' in the docstring. Also the description of px is not clear to me as amount suggests the total number of pixels (width x height). I suggest to use "tile_size" and describe it as the "width and height of a tile"

Reply:
Done

Reply:
it really is the amount of pixels, so I'll keep px. I'll update the docstring though to make it more clear.